packages/react-i18n/src/babel-plugin/test/fixtures/fallbackOnlyTranslation/translations/en.json

```diff
@@ -0,0 +1 @@
+../../adjacentTranslations/translations/en.json
```

Comment:
Not a fan of the symlinks tbh. I think it'd be better to keep these as regular files, even if there is a bit of duplication.